src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -7653,6 +7653,82 @@ class Goo: Attribute
                 Diagnostic(ErrorCode.ERR_NotAnAttributeClass, "X").WithArguments("X"));
         }
 
+        [Fact]
+        public void AmbiguousClassNamespaceLookup_Container()
```

Comment:
>AmbiguousClassNamespaceLookup_Container [](start = 20, length = 39)

Consider adding a WorkItem attribute

Reply:
These tests did not fail previously.

---
In reply to: [405651976](https://github.com/dotnet/roslyn/pull/43181#discussion_r405651976) [](ancestors = 405651976)